app/routes.py

```diff
@@ -0,0 +1,66 @@
+import os.path
+# using werkzeug.exceptions in order to get rid of pyramid
+# from pyramid.httpexceptions import HTTPBadRequest and other flask stuff
+# from pyramid.view import view_config
+# from pyramid.response import Response
+from flask import Response
+import six  # for compatibility Python 2 / 3
+from PIL import Image
+from app import app
+from app.helpers import check_color_channels
+from app.helpers import check_path
```

Comment:
The comment about pyramid is not necessary, it should be removed.
Although import should be grouped as defined in the guideline: https://github.com/geoadmin/doc-guidelines/blob/master/PYTHON.md#5-imports-formatting

Reply:
Comment removed. Order of imports will be re-checked, once pylint runs without errors